src/interdiction/Interdiction.jl

```diff
@@ -0,0 +1,249 @@
+__precompile__(true)
+module Interdiction
+
+using LightGraphs
+using JuMP
+using MathProgBase
+
+import MathProgBase.SolverInterface.AbstractMathProgSolver,
+       JuMP.UnsetSolver
+
+export interdiction_flow, ⪷,
+       NetworkInterdictionProblem, AdaptiveFlowArcProblem, AdaptiveFlowPathProblem,
+       MultilinkAttackAlgorithm, BilevelMixedIntegerLinearProgram
```

Comment:
I'm back from my trip now. I did the test locally again, there is no trouble.

Travis reports that the variable `multiroute_flow` is not defined. So this is definitively an import problem. Cf my post above.